src/components/30-organisms/table-sortable/ui.test.js

```diff
@@ -0,0 +1,217 @@
+import { Selector } from 'testcafe';
+
+fixture('Table Sortable - basic functionality').page(
+  'http://localhost:9999/iframe.html?id=organisms-table--table-sortable-default'
```

Comment:
take host and port from 'env'. It exists.

Reply:
button is also using loclahost. I will fix it but also button needs a fix here

Reply:
and the scaffolding script too

Reply:
done